onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -91,17 +106,40 @@ def testSetProvidersWithOptions(self):
             option['trt_int8_calibration_table_name'] = calib_table_name
             int8_use_native_calibration_table = "true"
             option['trt_int8_use_native_calibration_table'] = int8_use_native_calibration_table
+            dla_enable = "true"
```

Comment:
how can set these options  (DLA, fp16, int8) to true without checking if the hw supports it. wouldn't it throw error?
if it doesn't , then isn't that a problem?